Makefile

```diff
@@ -21,6 +21,9 @@ default: it
 .8.0:
 	$(NROFF) -man $< >$@
 
+%.o: %.c compile
```

Comment:
A slightly less readable but more portable way would be `.c.o:`, but I doubt this would support adding `compile` as dependency of `.o`.